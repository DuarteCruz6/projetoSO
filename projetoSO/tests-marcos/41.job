WRITE [(v2,28)(e1,86)(n1,82)]
READ [v2]
READ [v2]
READ [v2]
WRITE [(p7,8)(w1,27)(f5,20)(c3,12)(k2,16)]
READ [v2]
DELETE [k2,n1,f5,e1]
WRITE [(d6,9)(m1,16)(w5,6)(s7,65)(q2,98)]
WRITE [(a8,43)(a2,11)]
READ [q2]
READ [w5]
DELETE [d6,s7]
READ [w5]
WRITE [(p2,65)(j3,35)(q9,84)(t2,1)]
WRITE [(i5,8)(o6,97)(n1,9)]
DELETE [p7,a2,q2,m1,i5]
WRITE [(t5,68)(k3,37)(p8,58)]
DELETE [w1,j3]
