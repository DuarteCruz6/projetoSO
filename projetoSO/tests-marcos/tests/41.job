WRITE [(m9,83)(v9,3)(c8,93)(o7,84)]
WRITE [(h9,54)(v5,75)(b0,68)(h4,94)]
READ [v5]
DELETE [c8,v5,h4,o7,h9]
BACKUP
READ [b0]
READ [b0]
DELETE [v9,b0]
BACKUP
WRITE [(j1,25)(v9,77)]
WRITE [(w5,43)(e7,61)]
DELETE [v9,e7,w5]
DELETE [m9,j1]
BACKUP
BACKUP
WAIT 1000
WRITE [(s7,3)]
WRITE [(m6,85)(v9,55)]
WRITE [(w1,65)]
WRITE [(i8,18)(o0,91)(g8,88)(y5,49)]
READ [y5]
WRITE [(i4,18)(o4,24)(j6,61)]
WRITE [(v0,20)(a2,74)(v8,13)(n3,92)(k0,13)]
READ [o4]
BACKUP
WRITE [(l4,64)(z7,83)(x2,96)]
DELETE [k0]
DELETE [l4]
WRITE [(l6,32)]
