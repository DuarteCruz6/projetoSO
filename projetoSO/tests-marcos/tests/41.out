[(v5,75)]
[(b0,68)]
[(b0,68)]
[(y5,49)]
[(o4,24)]
